ios/browser/api/ads/brave_ads.h

```diff
@@ -56,6 +56,10 @@ OBJC_EXPORT
 /// Returns |true| if the ads service is running otherwise returns |false|.
 - (BOOL)isServiceRunning;
 
+// Returns |true| if always run the ads service, even if Brave Private Ads are
+// disabled.
```

Comment:
Should be `///` (three slashes) for documentation comments (Xcode picks them up)

Reply:
Fixed